src/core/database.cpp

```diff
@@ -819,6 +819,8 @@ bool Database::CreateAllTables()
 	CheckSuccess(success);
 	success = CreateTemplateMatchingResultsTable();
 	CheckSuccess(success);
+	success = CreateMovieMetadataTable();
+	CheckSuccess(success);
 
 	return success;
```

Comment:
@timothygrant80 I'm not sure what the return value is used for, but this is only going to report on the final value for success, right? Should there be an 'allPassed' or something?

Reply:
CheckSuccess is a macro, it will immediately return false from the function if anything fails.  i guess it might be more obvious if the final line said return true, but the effect is the same.